[
    {
        "id": 510351282,
        "name": "CVE-2015-10141",
        "full_name": "D3Ext\/CVE-2015-10141",
        "owner": {
            "login": "D3Ext",
            "id": 67125397,
            "avatar_url": "https:\/\/avatars.githubusercontent.com\/u\/67125397?v=4",
            "html_url": "https:\/\/github.com\/D3Ext",
            "user_view_type": "public"
        },
        "html_url": "https:\/\/github.com\/D3Ext\/CVE-2015-10141",
        "description": "POC exploit for CVE-2015-10141",
        "fork": false,
        "created_at": "2022-07-04T12:30:17Z",
        "updated_at": "2025-08-11T15:31:04Z",
        "pushed_at": "2025-08-11T15:11:04Z",
        "stargazers_count": 30,
        "watchers_count": 30,
        "has_discussions": false,
        "forks_count": 9,
        "allow_forking": true,
        "is_template": false,
        "web_commit_signoff_required": false,
        "topics": [
            "cve-2015-10141",
            "python"
        ],
        "visibility": "public",
        "forks": 9,
        "watchers": 30,
        "score": 0,
        "subscribers_count": 0
    }
]